assets/evaluators/builtin/task_completion/spec.yaml

```diff
@@ -11,6 +11,8 @@ initParameterSchema:
   properties:
     deployment_name:
       type: "string"
+    threshold:
```

Comment:
does this even accept threshold? https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_task_completion/_task_completion.py

If so, add constraints please